docker-compose/technical/grafana/dashboards/pods_supervision.json

```diff
@@ -1126,13 +1126,213 @@
       ],
       "type": "table"
     },
+    {
```

Comment:
After discussing with @SlimaneAmar , these metrics should be added to the table "Pods info and restarts".
total connections => current threads
active connections => busy threads

Reply:
there should be no timeseries for now